library/openshift_provision.py

```diff
@@ -112,7 +112,7 @@ def __init__(self, module):
                 self.oc_cmd += ['--insecure-skip-tls-verify='+connection['insecure_skip_tls_verify']]
 
     def merge_dict(self, merged, patch, overwrite=True):
-        for k, v in patch.iteritems():
+        for k, v in patch.items():
```

Comment:
I intentionally used `items` rather than `iteritems` for python 3 compatibility. If we were to switch to iteritems then we should also use python six. My opinion was that optimization in this case does not seem worth the trouble.